examples/ui/text.rs

```diff
@@ -26,14 +32,20 @@ struct FpsText;
 struct ColorText;
 
 fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    let text_content = if cfg!(target_arch = "wasm32") {
```

Comment:
Noobie question, what is the reason for this check? Doesn't wasm32 support emojis or something like that?

Also, (purely personal preference), I would use match instead here. Never liked rust's ternary syntax.

```rust
let text_content = match cfg!(target_arch = "wasm32") {
    true => "hello\nbevy!"
    false => "hello\nbevy! 😌"
};
```

Reply:
It needs access to file system, so on wasm32 it won't do anything... won't load system font so emoji will be rendered improperly... unless user loads font explicitly that support emoji for wasm32.